.eslintrc.js

```diff
@@ -17,29 +14,23 @@ module.exports = {
     jest: true
   },
   settings: {
-      react: {
-          version: '16.0'
-      },
-      'import/resolver': {
-          node: {
-            paths: ['./src']
-          }
+    'import/resolver': {
+      node: {
+        paths: ['./src']
       }
+    }
   },
   plugins: [
-    'react',
-    'react-hooks',
     'jsdoc',
-    'jsx-a11y',
-    'sort-destructure-keys'
+    'sort-destructure-keys',
+    'node'
   ],
   rules: {
     'brace-style': ['error', 'stroustrup'],
     'comma-dangle' : [2, 'always-multiline'],
     'curly': [2, 'multi-line', 'consistent'],
     'no-unused-vars': [2, { 'ignoreRestSiblings': false }],
     'node/no-deprecated-api': 1,
-    'react-hooks/rules-of-hooks': 'error',
```

Comment:
Why was this taken out? Is it part of Airbnb already? Or was there a different reason?

Reply:
Yeah, it's already there: https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/rules/react-hooks.js#L15